packages/orchestrator-ui-components/src/components/WfoSearchBar/WfoSearchField.tsx

```diff
@@ -22,15 +24,20 @@ export const WfoSearchField = ({
         getFormFieldsBaseStyle,
     );
 
-    const queryIsValid = true; // Query validation turned of for now until ESQueries can be sent to the backend
+    const handleSearch = useRef((queryText: string) =>
```

Comment:
FMI. What's the benefit of using useRef here instead of useCallback?

Reply:
The reference keeps changing and triggers a renderloop. useCallback also requires a dependency array (just like useEffect)